public/js/script.js

```diff
@@ -32,6 +35,7 @@ function updateSelector(screen_data) {
     }
 }
 
+// FIXME does not corectly switch between screens
```

Comment:
haha, that's fixed now! So I'll add a TODO comment to the remove FIXME coment ;) :P